stories/virtru/fileListItem/progress.jsx

```diff
@@ -1,21 +1,27 @@
 import React, { useCallback, useState } from 'react';
 import { storiesOf } from '@storybook/react';
 
-import { FileListItem, Button } from '@';
+import { FileListItem, Button, Switch } from '@';
 
 import 'antd/dist/antd.less';
 
 storiesOf('virtru/fileListItem', module).add(
   'progress',
   () => {
+    const [loading, setLoading] = useState(true);
```

Comment:
Nit: We could also use [knobs](https://www.npmjs.com/package/@storybook/addon-knobs) which it looks like we have installed.